_config.yml

```diff
@@ -277,6 +277,9 @@ mathjax:
 # Gentie productKey
 #gentie_productKey:
 
+# youyan uid
```

Comment:
just fill in the `youyan_uid` in the `_config.yml` for adding youyan comment in your hexo